### DISCUSSION TITLE
Trying to write video file with mp4 crate, resulting file is not seekable.

### DISCUSSION THREAD

**Discussion by IntQuant** [ralfbiedert/openh264-rs#36] [General]
Hi,

I'm currently trying to write a stream of RgbImage's to mp4 file. It kinda works, but the resulting file is not seekable.

Versions:
```
mp4 = "0.13.0"
openh264 = "0.4.1"
image = "0.24.6"
```

[video.rs.zip](https://github.com/ralfbiedert/openh264-rs/files/11769211/video.rs.zip)

https://github.com/ralfbiedert/openh264-rs/assets/25219447/cc684750-2fc9-4462-9332-c04626ff3898

**Comment by IntQuant** [comment]
Looks like I forgot to stop creating tracks once one exists. The issue is still here tho.

**Comment by ralfbiedert** [comment]
I'm afraid that question is better asked in the mp4 (the crate) discussion forum, as seeking behavior is probably related to the container format.

**Comment by IntQuant** [comment]
I think I figured it out: IdrFrames have to be marked as sync. Thanks for the response.
